index.js

```diff
@@ -220,9 +219,9 @@ export async function setupPlugin({ config, global, jobs }) {
     global.dataPlaneUrl = config.dataPlaneUrl
 }
 
-export async function exportEvents(events, { global }) {
+export async function onEvent(event, { global }) {
     const payload = {
-        batch: events.map(constructRudderPayload),
+        ...constructRudderPayload(event),
```

Comment:
I'm a little surprised the API supports `batch: [...event...]` _and_ a single event's keys being placed directly in the payload?

Reply:
it might not 🤷‍♀️  this is a good spot

Reply:
I could just wrap that in a `[]`